src/AddBookForm.js

```diff
@@ -5,51 +5,67 @@ import useForm from 'react-hook-form';
 import { TextField, Button, Grid, createStyles, makeStyles } from '@material-ui/core';
 
 const mutation = gql`
-  mutation AddBook($title: String!, $author: String!) {
-    addBook(title: $title, author: $author) {
+  mutation AddBook($title: String!, $author: String!, $numPages: Int!) {
+    addBook(title: $title, author: $author, numPages: $numPages) {
       id
       title
       author
+      numPages
     }
   }
 `;
 
 const useStyle = makeStyles((theme) =>
   createStyles({
-    item: {
-      margin: theme.spacing(1),
+    root: {
+      width: '60%'
```

Comment:
Can we put commas at the end of each line.
Preferably we'd have automatic lint for this repo so we don't need these kinds of review comments.

Reply:
https://github.com/reshufflehq/apollo-gql-bookstore/issues/4